helm_deploy/values-test.yaml

```diff
@@ -0,0 +1,30 @@
+# Per environment values which override defaults in hmpps-assess-risks-and-needs/values.yaml
+
+generic-service:
+  replicaCount: 2
+
+  ingress:
+    host: assess-risks-and-needs-test.hmpps.service.justice.gov.uk
+    contextColour: green
+    annotations:
+      external-dns.alpha.kubernetes.io/aws-weight: "100"
+
+  env:
+    OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
+    COMMUNITY_API_BASE_URL: https://arns-and-delius-dev.hmpps.service.justice.gov.uk
+    OASYS_API_BASE_URL: https://t2-b.oasys.service.justice.gov.uk/eor/oasys
+    SENTRY_ENVIRONMENT: "dev"
```

Comment:
Should this value be `"test"` instead of dev? Same for `OAUTH_ENDPOINT_URL`

Reply:
Looks like it's not used in the application anywhere 🤷‍♂️

Reply:
Cool, happy to approve then